### PR TITLE
[osgearth] Change blend2d dependency to use the blend2d target 

### PR DESCRIPTION
**Pull request by BillyONeal**
and add missing dependency.

When I thought I fixed this the first time in https://github.com/microsoft/vcpkg/pull/15777 I thought `find_package(xxx REQUIRED)` meant that dependency was safe, but osgearth had their own findblend2d.cmake file which made the find_package succeed but which did not actually make the dependency required.

### CODE REVIEW

File: ports/osgearth/portfile.cmake [hunk 1]
```diff
@@ -51,30 +53,30 @@ vcpkg_configure_cmake(
         -DWITH_EXTERNAL_TINYXML=ON
 )
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 
 if (WIN32 AND (VCPKG_LIBRARY_LINKAGE STREQUAL dynamic))
```
Comment (dg0yt) [inline ports/osgearth/portfile.cmake:58]: `"dynamic"`

File: ports/osgearth/use-blend2d-target.patch [hunk 2]
```diff
@@ -0,0 +1,20 @@
+diff --git a/src/osgEarth/CMakeLists.txt b/src/osgEarth/CMakeLists.txt
+index 7cecdac..29bc05b 100644
+--- a/src/osgEarth/CMakeLists.txt
++++ b/src/osgEarth/CMakeLists.txt
+@@ -909,11 +909,11 @@ IF(FILEGDB_FOUND)
+     LINK_WITH_VARIABLES(${LIB_NAME} FILEGDB_LIBRARY)
+ ENDIF()
+ 
+-IF(BLEND2D_FOUND)
++#IF(BLEND2D_FOUND)
```
Comment (JackBoosY) [inline ports/osgearth/use-blend2d-target.patch:10]: So should we update patch `make-all-find-packages-required.patch` to modify
```
+find_package(Blend2D REQUIRED)
```
to
```
+find_package(Blend2D CONFIG REQUIRED)
```
instead?

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/osgearth/use-blend2d-target.patch (hunk 2 above)
Comment (dg0yt) [reply ports/osgearth/use-blend2d-target.patch:10]: IIUC `find_package(Blend2D CONFIG)` will set `Blend2d_FOUND`, not `BLEND2D_FOUND`.

Comment (JackBoosY) [reply ports/osgearth/use-blend2d-target.patch:10]: Also I think these changes should be merged into `make-all-find-packages-required.patch`.

**Review by JackBoosY — APPROVED** [review]

**Pull request merged**